README.md

```diff
@@ -316,6 +316,31 @@ setVerbose(True)
 setVerbose(False)
 ```
 
+
+### Output Files
+Batch will automatically handle your output files when the user assigns a file pattern and container url. 

```

Comment:
Probably don't want to just use 'container' since people may imply Docker Containers. Please use 'storage container' instead.